.github/workflows/release.yaml

```diff
@@ -0,0 +1,83 @@
+name: Release Flow
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-packages:
+    name: Push Packages
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-versions }}
+      - uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Pushing Packages"
+              cd /work
+              export AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1)
+              autonomy init --reset --author $AUTHOR --ipfs --remote
+              autonomy push-all
+  publish-images:
+    name: Publish Docker Images
+    runs-on: ${{ matrix.os }}
+    needs:
+      - "publish-packages"
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up tag and vars
+        uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Setting Tag Images"
+              cd /work
+              echo "VERSION=0.8.0" > env.sh
+              echo AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1) >> env.sh
+              echo SERVICE=$(grep 'service' packages/packages.json | awk -F/ '{print $3}' | head -1) >> env.sh
+              echo DEFAULT_IMAGE_TAG=$(cat packages/packages.json | grep agent | awk -F: '{print $2}' | tr -d '", ') >> env.sh
```

Comment:
note the trick from the main repos to get the version doesnt work as these are not packages